### DISCUSSION TITLE
createTexture is not a function

### DISCUSSION THREAD

**Discussion by lucadalli** [google/model-viewer#2647] [Q&A] [answered]
I am trying to replicate the ["Create Textures" example](https://modelviewer.dev/examples/scenegraph/#createTexturesExample) locally, but I am faced with an error that says `modelViewerTexture.createTexture is not a function`. 

All mentions of `createTexture` in the [un-minified bundle](https://unpkg.com/@google/model-viewer@1.7.2/dist/model-viewer.js) seem to be coming from three.js

Was this part of the docs released prematurely? Will `createTexture` be part of the next release?

I'm using version 1.7.2.

**Comment by elalish** [comment, marked as answer]
Yeah, we should really find a more prominent place to note this: the docs are served off master, so they get updated before release. I'm planning to get v1.8 out next week which will include this.
